Source/Core/DiscIO/NANDImporter.cpp

```diff
@@ -23,16 +25,19 @@ NANDImporter::NANDImporter() = default;
 NANDImporter::~NANDImporter() = default;
 
 void NANDImporter::ImportNANDBin(const std::string& path_to_bin,
-                                 std::function<void(size_t, size_t)> update_callback)
+                                 std::function<void()> update_callback)
 {
   m_update_callback = std::move(update_callback);
 
   if (!ReadNANDBin(path_to_bin))
     return;
 
   const std::string nand_root = File::GetUserPath(D_WIIROOT_IDX);
+  m_nand_root_length = nand_root.length();
```

Comment:
If I don't want to store the entire string and calculate the size everytime, then yes. It makes strings like `D:/Dev/Projects/dolphin/Binary/x64/User/Wii/title/00000001/00000050` into `/title/00000001/00000050`. As you might be able to see, it reveals folder names and gets quite long in most cases